.github/workflows/ci.yml

```diff
@@ -17,16 +17,12 @@ jobs:
       - name: Build libROM
         run: |
             mkdir ${GITHUB_WORKSPACE}/build
-            cd ${GITHUB_WORKSPACE}/build
             export CC=mpicc
             export CXX=mpicxx
-            cmake -DCMAKE_BUILD_TYPE=Debug ..
-            make
-            cmake -DCMAKE_BUILD_TYPE=Optimized ..
-            make
+            scripts/compile.sh -m
       - uses: ./.github/workflows/run_tests
   mac:
-    runs-on: macos-latest
+    runs-on: macos-12
```

Comment:
I just tried this to see if it would work. This doesn't have to be macos-12. It could go back to macos-latest.